app/models/registration.rb

```diff
@@ -517,4 +528,53 @@ def competition_events_changed?
   def serializable_hash(options = nil)
     super(DEFAULT_SERIALIZE_OPTIONS.merge(options || {}))
   end
+
+  def attempt_auto_accept
+    return false if Rails.env.production? && EnvConfig.WCA_LIVE_SITE?
+
+    failure_reason = auto_accept_failure_reason
+    if failure_reason.present?
+      log_auto_accept_failure(failure_reason)
+      return false
+    end
+
+    update_payload = build_auto_accept_payload
+    if Registrations::RegistrationChecker.apply_payload(self, update_payload).valid?
+      update_lanes!(
+        update_payload,
+        'Auto-accept',
+      )
+      true
+    else
+      log_auto_accept_failure(accept_registration.errors)
```

Comment:
What is `accept_registration` and where does it come from?

Reply:
It was left orphaned during a refactor - fixed now, but this `else` block isn't under test until the registration model validates status